src/games/strategy/engine/framework/GameRunner2.java

```diff
@@ -241,23 +242,24 @@ private static String getValue(final String arg) {
   }
 
   public static void setupLookAndFeel() {
-    try {
-      UIManager.setLookAndFeel(getDefaultLookAndFeel());
-      // FYI if you are getting a null pointer exception in Substance, like this:
-      // org.pushingpixels.substance.internal.utils.SubstanceColorUtilities
-      // .getDefaultBackgroundColor(SubstanceColorUtilities.java:758)
-      // Then it is because you included the swingx substance library without including swingx.
-      // You can solve by including both swingx libraries or removing both,
-      // or by setting the look and feel twice in a row.
-    } catch (final Throwable t) {
-      System.out.println("LOOK AND FEEL exception: " + t.getMessage());
-      if (!GameRunner.isMac()) {
-        try {
-          UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
-        } catch (final Exception e) {
+    SwingAction.invokeAndWait(() -> {
```

Comment:
Can this block be cleaned up some to make it more readable? Lots of nesting makes this hard to follow.